reporter/src/main/scala/com/typesafe/tools/mima/lib/analyze/method/MethodChecker.scala

```diff
@@ -37,7 +37,9 @@ private[analyze] abstract class BaseMethodChecker extends Checker[MemberInfo, Cl
 
 private[analyze] class ClassMethodChecker extends BaseMethodChecker {
   def check(method: MemberInfo, inclazz: ClassInfo): Option[Problem] = {
-    if (method.isDeferred)
+    if (method.nonBridge && method.nonAccessible)
```

Comment:
Yeah, but there is still something in my head that doesn't click. Let's have a quick chat about this if you can. Also, #36 is related.